### ISSUE TITLE
Reading card data

### ISSUE THREAD

**Issue by MikeyAlder** [techniq/node-pn532#1]
How do you read card data? or is this not developed yet :-/

**Comment by techniq** [comment]
I figured out the 5 bytes at the head of the data pages (they are for lock information).  I found the NFC Forum Type 2 technical specification which gave me great information on the format (straight from the source, instead of culminated from a few different internet resources).  You can grab the specification documents after signing a [license agreement](http://members.nfc-forum.org/specs/spec_license).

**Comment by techniq** [comment]
I just pushed my changes that implement tag reading and writing.  I also updated some method and event names, so you'll want to take a look at the README (`card` event -> `tag` event, `readPassiveTargetId()` -> `scanTag()`, etc).  Currently the reading and writing for tag data working on my NTAG203, but it should work on any card type with 4 byte pages (other NTAG* tags, Mifare Ultralight).  It definitely doesn't work on Mifare Classic tags, as they use 16-byte blocks, and require authentication that I've looked into, but haven't resolved yet.  I've closed this issue but will open one for Mifare Classic support.  I also plan to implement tag emulation soon for a project I'm working on (have the PN532 act as a tag / scanned by another device, instead of as a scanner).

Let me know how it works for you.  You'll need to pull from source as I haven't pushed an update to NPM yet, but hopefully will do so in a few days.
